subsys/spm/spm.c

```diff
@@ -355,7 +360,7 @@ static void spm_configure_ns(const tz_nonsecure_setup_conf_t
 	/* Set non-banked exceptions to target Non-Secure */
 	tz_nbanked_exception_target_state_set(0);
 	/* Don't allow Non-Secure firmware to issue System resets. */
-	tz_nonsecure_system_reset_req_block(1);
+	tz_nonsecure_system_reset_req_block(SPM_BLOCK_REBOOT);
```

Comment:
Could we use `IS_ENABLED(CONFIG_SPM_SERVICE_REBOOT)` directly here?

Reply:
Would assume so. Changed.